src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static final FigureName figureName = FigureName.RECTANGLE;
```

Comment:
Don't use static fields for properties that are supposed to be different for each object instance. The `figureName` should not be static because it's the same for all instances of the class but it's not a configuration property or a constant.